core/src/main/res/values/strings.xml

```diff
@@ -229,6 +229,8 @@
     <string name="download_type_media">Media file</string>
     <string name="download_type_image">Image</string>
     <string name="download_request_error_dialog_message_prefix">An error occurred when trying to download the file:\u0020</string>
+    <string name="error_label_error_show_null_podcast">Error</string>
```

Comment:
You can delete that line and use `R.string.error_label` instead. That string is already defined in the file, so you don't need to add the string again.